client/src/components/navbar.css

```diff
@@ -0,0 +1,3 @@
+#links:hover {
```

Comment:
You should be able to move this styling into the useStyles in Navbar.js

Reply:
Thanks for the feedback. I've fixed the four issues you've mentioned and pushed the changes. Let me know if anything else stands out!